README.md

```diff
@@ -1731,6 +1730,17 @@ Disabling Auto-Init means that you _must_ explicitly call `terragrunt init` prio
 
 If Auto-Init is disabled, and terragrunt detects that `terraform init` needs to be called, then terragrunt will fail.
 
+### Auto-Retry
+
+_Auto-Retry_ is a feature of `terragrunt` that will automatically address situations where a `terraform` command needs to be re-run.
+
+Terraform can fail with transient errors which can be addressed by simply retrying the command again. In the event `terragrunt` finds one of these errors, the command will be re-run again automatically.
+
+In the event of a module or provider update, `terraform` will fail with a message to run `terraform init`, in this case `terragrunt` will re-run the `init` command and then retry the command. 
```

Comment:
Will Terragrunt re-run only on `init` or any command that fails (e.g., `apply`, `destroy`)?

Reply:
Also, how many times will Terragrunt retry? Once? Infinite? Is there a pause before retrying?